src/main/java/ca/bc/gov/chefs/etl/forms/pcd/hrRecords/processor/PcdHRRecordsApiResponseProcessor.java

```diff
@@ -32,7 +33,9 @@ public void process(Exchange exchange) throws Exception {
         String payload = exchange.getIn().getBody(String.class);
         ObjectMapper mapper = new ObjectMapper();
 
-        List<Root> hrRecordsModels = mapper.readValue(payload, new TypeReference<List<Root>>() {
+        String fixedPayload = JsonUtil.fixPcnNameObject(payload);
```

Comment:
I think it's fine to just re-assign payload. That's what most of the processors do.